web/app/scripts/views/record/list-controller.js

```diff
@@ -18,7 +19,30 @@
         function initialize() {
             RecordState.getSelected().then(function(selected) { ctl.recordType = selected; })
                 .then(loadRecordSchema)
-                .then(loadRecords)
```

Comment:
That wasn't the source of the double-loading; that was [here](https://github.com/WorldBank-Transport/DRIVER/pull/142/files#diff-c2eba903e8631b2cfe3f6bef2b095e34R65).